sklearn/pipeline.py

```diff
@@ -638,6 +642,12 @@ def _get_name(name, est):
                             name_details=name_details,
                             dash_wrapped=False)
 
+    def _more_tags(self):
+        more_tags = {}
+        for _, estimator in self.steps:
+            more_tags.update(estimator._more_tags())
+        return more_tags
```

Comment:
I don't think we should implement this logic for this PR.

We should focus on `_estimator_type` only, so please only override the tags for that key, based on the last step

Something like

```py
return {'estimator_type': self.steps[-1][1]._get_tags()['estimator_type']}
```

Reply:
I am +1 about focusing only in the `estimator_type` tag for this PR.

WDYT @thomasjpfan?